cmd/exec.go

```diff
@@ -38,18 +41,30 @@ func NewKoolExec() *KoolExec {
 		*newDefaultKoolService(),
 		&KoolExecFlags{false, []string{}, false},
 		environment.NewEnvStorage(),
-		builder.NewCommand("docker-compose", "exec"),
+		compose.NewDockerCompose("exec"),
 	}
 }
 
 // Execute runs the exec logic with incoming arguments.
 func (e *KoolExec) Execute(args []string) (err error) {
+	if asuser := e.env.Get("KOOL_ASUSER"); asuser != "" {
```

Comment:
how much cost (time) it comes? maybe we could cache this somehow...

Reply:
during a program execution, golang copies the entire environment variables list into its memory only once. So, I guess the only cost we have here is to lock the `map` and return the value.

Reply:
I believe Daniel misplaced the comment - the question was targeted at the `cat /etc/passwd` extra `exec` call that we will make now. And although we could think in some kind of cache, that would introduce the whole project-specific-storage of runtime configuration that we do not have currently, so I would leave with the overhead for now, and only consider this in the future.

Reply:
@fabriciojs I got it! I agree that we should cache `passwd` as well. But for me is ok doing it in a next step.